client/core/trade.go

```diff
@@ -804,15 +808,15 @@ func (t *trackedTrade) shouldBeginFindRedemption(match *matchTracker) bool {
 }
 
 // tick will check for and perform any match actions necessary.
-func (t *trackedTrade) tick() (assetMap, error) {
+func (c *Core) tick(t *trackedTrade) (assetMap, error) {
```

Comment:
I've elevated a handful of methods to `*Core` methods. I didn't move them, because it would make the diff ridiculous, but I'll do some re-arranging in another PR.

Reply:
The main reason being that the `finalize*` functions now require calling `(*Core).resolveMatchConflicts` and on to one of the `conflictResolvers`?  I'm prying because each one of the `conflictResolvers` seems to have the `*Core` input argument unused.  I replaced `(c *Core...` with `(_ *Core...` on each one and there were no compilation issues.  Perhaps the `type matchConflictResolver func(*Core, *dexConnection, *trackedTrade, *matchTracker, *msgjson.MatchStatusResult)` can be redefined without a `*Core`?

Reply:
Doesn't this mean that redefining the receiver isn't necessary any more, noting the c.notify is replaceable because the notify function is in the trackedTrades too?  Really though, I have no objection to the change.  Just want to understand the considerations.